pom.xml

```
@@ -1552,6 +1552,9 @@
               <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
               <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
             </manifest>
+            <manifestEntries>
+              <Automatic-Module-Name>com.puppycrawl.tools.checkstyle</Automatic-Module-Name>
```

Can we change it to something like:
````suggestion
              <Automatic-Module-Name>${project.groupId}.${project.artifactId}</Automatic-Module-Name>
````

---

Sounds like a good idea. Please keep in mind, that the Java module name may be the same as the Maven identifier but does not need to be. Often Maven identifiers have a dash. This is not allowed in a module name. But in your case, it will work of course.